### DISCUSSION TITLE
Which version of Chia must I install to farm?

### DISCUSSION THREAD

**Discussion by Tavanaie** [Chia-Network/chia-blockchain#422] [General]
I have installed latest Chia block chain software and now the network size raised alot ,  I was reading release notes , and it shown me that this version is test!
So can I farm in this version? or I must use version 0.11 instead?

**Comment by hoffmang9** [comment]
All coins are testnet monopoly money until mainnet which we expect to occur at the end of the year.

**Comment by Tavanaie** [comment]
Can you describe more please?
All Chia farmed until the end of the year (mainnet release) will be lost?

**Reply by hoffmang9** [reply]
Yes - this is testnet and not mainnet chia.

**Reply by Tavanaie** [reply]
Oh,  But I have read that this is the initiative coin farming phase , just before the final release , so there is no gain for farming now?

**Reply by hoffmang9** [reply]
When mainnet launches there will be a period when only farming rewards are given. This is to allow the chain to stabilize and for farmers to get the first rewards. That period is likely 4 to 6 weeks but only begins once mainnet launches. Other than contests (and we plan to run another soon) for top farmers during a period, there is no direct incentive to farm right now.

**Comment by Tavanaie** [comment]
Thanks alot
Can you please tell me that if we create plots for farming by beta version of Chia block chain , those plots will be usable at mainnet or not?

**Reply by hoffmang9** [reply]
Starting with Beta 8 any k=32 or larger plot will work with mainnet. K=31 May or May not work - we’re not sure yet. K=30 or smaller will be too small.

**Comment by Kamillewinter** [comment]
Unfinished blocks    Can you farming?

**Reply by Kamillewinter** [reply]
I can't farm, what to do?

**Reply by Tavanaie** [reply]
What I know is that farming process is based on Luck
When you connect,  your luck  to gain chia is about 4000 multiplied by your plot space , divided to total size of the network , so maybe you win first chia immediately , or maybe it take long depending on your and network plot space

**Comment by BrandtH22** [comment]
Chia launched mainnet more than 3 years ago and the latest releases can be downloaded from https://www.chia.net/downloads/.
Feel free to open a new discussions if any questions arise.
